docs/stdlib/json.rst

```diff
@@ -405,9 +403,9 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
         ... );
         {'{"a": 10, "b": 20}'}
 
-    ``empty_treatment`` is an enumeration responsible for the behavior of the
-    function if an empty set is passed to ``new_value``. It contains one of
-    the following values:
+    ``empty_treatment`` is an enumerable responsible for the behavior of the
+    function if an empty set is passed to ``new_value``. This will contain one
+    of the following values:
```

Comment:
I believe the enumeration -> enumerable change changes the meaning, but I'm not sure the original is 100% clear either.

@vpetrovykh Could you do a sanity check on this rewrite?

```suggestion
    *empty_treatment* can be any value from the `JsonEmpty` enumeration. It
    defines the behavior of the function if an empty set is passed as
    *new_value*. `JsonEmpty` contains these values:
```